apps/web/src/app/settings/layout.tsx

```diff
@@ -10,7 +10,7 @@ export default async function SettingsLayout({
   const session = await auth();
   
   if (!session?.user) {
-    redirect("/auth/signin");
+    redirect("/api/auth/signin");
```

Comment:
we should use a constant for this. maybe a routes.ts file for many of these or similar.